api/SVGURIReference.json

```
@@ -12,7 +12,7 @@
             "version_added": null
           },
           "edge": {
-            "version_added": null
+            "version_added": false
```

```suggestion
            "version_added": null
```

`SVGURIReference` is part of the problem in https://github.com/mdn/browser-compat-data/issues/7752 and should be removed from BCD/MDN, but it's not technically true that it's not supported, because the `href` attributes is supported on various interfaces. Reverting this back to null so that it's an all-null entry and less to wonder about when it's removed.